bin/bench-exec-one.sh

```diff
@@ -102,8 +102,8 @@ bench_rts_opts_specific () {
 
 if test "$USE_GAUGE" -eq 0
 then
-  SUPER_QUICK_OPTIONS=""
-  QUICKER_OPTIONS=""
+  SUPER_QUICK_OPTIONS="--stdev 100"
```

Comment:
Why not use `Infinity` instead of 100?

Reply:
for Infinity  It hangs forever. As per docs it is "Target relative standard deviation of measurements in percents" so max pct is 100.

Reply:
Deviation can be more than 100%. Not sure why it hangs with `Infinity`. Did you try a very large value e.g. 10000? Did you use the latest tasty-bench from github? 

@bodigrim is this tested with `Infinity`?

Reply:
`tasty-bench` HEAD supports `--stdev Infinity`, but earlier (released) version does not handle it correctly and hang indeed. Sorry for that.

@harendra-kumar I can make a new release from `tasty-bench` HEAD, if desirable.